sdk/textanalytics/azure-ai-textanalytics/samples/README.md

```diff
@@ -1,31 +1,34 @@
 ---
-topic: sample
+page_type: sample
 languages:
   - python
 products:
   - azure
   - azure-ai-textanalytics
+urlFragment: cognitiveservices-textanalytics-samples
 ---
 
 # Samples for Azure Text Analytics client library for Python
 
 These code samples show common scenario operations with the Azure Text Analytics client library.
-The async versions of the samples (the python sample files appended with `_async`) show asynchronous operations 
-with Text Analytics and require Python 3.5 or later. 
+The async versions of the samples require [Python 3.5][python_35] or later.
```

Comment:
It feels weird to specifically link to 3.5 here. I would link to the latest version of python or don't link at all

Reply:
Yeah agreed, I think I went to trigger happy with links